cli/azd/.vscode/cspell.yaml

```diff
@@ -40,8 +47,7 @@ overrides:
       - azuredeps
   - filename: internal/appdetect/java.go
     words:
-      - springframework
-      - eventhubs
+      - chardata
```

Comment:
Hi, @haoozhang , could you please confirm that this word is necessary? Can we change to another word?